core/common/src/main/java/alluxio/metrics/MetricKey.java

```diff
@@ -1010,8 +1017,16 @@ public MetricKey build() {
           .build();
   public static final MetricKey WORKER_BYTES_READ_REMOTE =
       new Builder("Worker.BytesReadRemote")
+          .setDescription("Total number of bytes read from "
```

Comment:
It seems the meaning of this metrics has been changed after this PR? So would you like to give a new name of this metrics?